### PR TITLE
added prominence/width parameters to docs

### PR DESCRIPTION
**Pull request by NikitaPoly**

### CODE REVIEW

**Comment by jperson** [comment]
I might be wrong but I think you'll need to bump the version too otherwise I don't think this will publish

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,5 +1,9 @@
 Changelog for Pulse3D
 =====================
+0.24.7 (2922-08-09)
```
Comment (tannermpeterson) [inline CHANGELOG.rst:3]: 2022*

**Comment by tannermpeterson** [comment]
You added an entry for the new version in the changelog, but you still need to bump the version in setup.py in order to publish

**Comment by tannermpeterson** [comment]
You might not actually need to bump the version at all though, I think readthedocs will automatically pick up the changes to the docs once you merge this into main. I'll double check

**Review by tannermpeterson — APPROVED** [review]

**Pull request merged**